go/flags/endtoend/vttablet.txt

```diff
@@ -267,6 +267,7 @@ Usage of vttablet:
       --serving_state_grace_period duration                              how long to pause after broadcasting health to vtgate, before enforcing a new serving state
       --shard_sync_retry_delay duration                                  delay between retries of updates to keep the tablet and its shard record in sync (default 30s)
       --shutdown_grace_period duration                                   how long to wait (in seconds) for queries and transactions to complete during graceful shutdown. (default 0s)
+      --signal-schema-change-reload-timeout duration                     query server schema change signal reload timeout, this is how long to wait for the signaled schema reload operation to complete before giving up (default 30s)
```

Comment:
Can we call this flag `schema-change-reload-timeout`? I know this is a nit, but I don't like the signal in the name. With a more generic name, we can use the same flag for the schema tracking in `schema.Engine` which is being augmented to in https://github.com/vitessio/vitess/pull/13121

Reply:
It makes sense!
Fixed by [a0709c8](https://github.com/vitessio/vitess/pull/13111/commits/a0709c8ef8c5ec0005cc6929e3529a2943dc1bce).